README.md

```diff
@@ -40,7 +48,9 @@ In an AJAX POST request, what does the argument of the `.done()` callback - in t
   });
 ```
 ### Question 6
-Suppose you had the following form in your HTML file. Use jQuery to write a single line of code that takes whatever the user entered in the textbox and saves it to the variable `user_input`. 
+Suppose you had the following form in your HTML file. Use jQuery to write a single line of code that takes whatever the user entered in the textbox and saves it to the variable `user_input`.
+
+var user_input = $('#submit');
```

Comment:
This code would select the submit button. You want to select the text box, and then use the .val() method to get what the user typed into it. So that would be `var user_input = $('#color').val()`.